lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -347,4 +346,11 @@ module.exports = {
     const split = filterName.split('-');
     return split[split.length - 2];
   },
+
+  getFunctionNameFromDestinationArn(destinationArn) {
+    // Destination arn format:
+    // arn:{partition}:lambda:{region}:{accountId}:function:{functionName}
+    const split = destinationArn.split(':function:');
```

Comment:
Is that always going to be function name? What about functions with aliases e.g. with `provisionedConcurrency`?

Reply:
That should be unconsidered edge case 😢 Its quiet non-intuitive thing, so I changed to use subscription filter name to get "describeStackResources" instead of lambda function name